Wikipedia/Code/WMFWelcomeAnalyticsViewController.m

```diff
@@ -33,20 +34,42 @@ - (void)viewDidLoad {
         self.toggle.on                                                           = NO;
         [[BITHockeyManager sharedHockeyManager] crashManager].crashManagerStatus = BITCrashManagerStatusAlwaysAsk;
     }
+
+    [self.buttonCaretLeft setTintColor:[UIColor wmf_blueTintColor]];
+    [self.buttonCaretLeft wmf_setButtonType:WMFButtonTypeCaretLeft];
 }
 
-- (IBAction)toggleAnalytics:(id)sender {
-    if ([(UISwitch*)sender isOn]) {
+- (IBAction)toggleAnalytics:(UISwitch*)sender {
+    if ([sender isOn]) {
         [[BITHockeyManager sharedHockeyManager] crashManager].crashManagerStatus = BITCrashManagerStatusAutoSend;
         [[NSUserDefaults standardUserDefaults] wmf_setSendUsageReports:YES];
+        
+        dispatchOnMainQueueAfterDelayInSeconds(1.4, ^{
```

Comment:
@nirzar @montehurd agree with @bgerstle here. Users just don't expect a view to disappear because the flip a switch. 

If we really want it to disappear on interaction then there show be 2 buttons - and tapping yes/no would then dismiss.